perf/Cargo.toml

```diff
@@ -21,22 +21,29 @@ log = { workspace = true }
 rand = { workspace = true }
 rayon = { workspace = true }
 serde = { workspace = true }
+solana-clock = { workspace = true, optional = true }
 solana-frozen-abi = { workspace = true, optional = true, features = [
     "frozen-abi",
 ] }
 solana-frozen-abi-macro = { workspace = true, optional = true, features = [
     "frozen-abi",
 ] }
 solana-hash = { workspace = true }
+solana-keypair = { workspace = true, optional = true }
+solana-message = { workspace = true }
 solana-metrics = { workspace = true }
 solana-packet = { workspace = true, features = ["bincode"] }
-solana-program = { workspace = true, default-features = false }
 solana-pubkey = { workspace = true, default-features = false }
 solana-rayon-threadlimit = { workspace = true }
 solana-sdk = { workspace = true, optional = true }
```

Comment:
Looks like this got missed -- can you remove it and the `solana-sdk/frozen-abi` feature activation? I tried it locally and it worked.